### PR TITLE
Added SystemdDnsmasqServiceConf method to the systemd_config parser

### PR DESCRIPTION
**Pull request by rasrivas-redhat**
Signed-off-by: Rahul Srivastava <rasrivas@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This is an enhancement PR to check for the `/bin/systemctl cat dnsmasq.service` config

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

**Review by bfahr — APPROVED** [review]

**Pull request merged**
